requirements.txt

```diff
@@ -19,3 +19,4 @@ types-requests
 types-setuptools>=57.0.0
 ruff>=0.0.259; python_version > "3.6"
 isort>=5.12.0,<6.0; python_version > "3.7"
+spacy==3.5.1
```

Comment:
We don't want to have a dependency on `spacy` in `weasel` though? It shouldn't be here just for the test suite...

Reply:
See my comment [here](https://github.com/explosion/weasel/pull/15#discussion_r1157013542). We can definitely remove this.